src/ui/ui.css

```diff
@@ -45,18 +47,13 @@ swg-loading-container {
 }
 
 swg-container,
```

Comment:
Could you move all these `display:block` to the very top?

Reply:
Done. Moved after body {...}